Xamarin.Forms.Platform.UAP/FontExtensions.cs

```diff
@@ -105,16 +107,27 @@ public static FontFamily ToFontFamily(this string fontFamily)
 			return font;
 		}
 
+		static string FindFontFamilyName(string fontFile)
+		{
+			using (var fontSet = new CanvasFontSet(new Uri(fontFile)))
+			{
+				if (fontSet.Fonts.Count == 0)
+					return null;
+
+				return fontSet.GetPropertyValues(CanvasFontPropertyIdentifier.FamilyName).FirstOrDefault().Value;
```

Comment:
Will `FirstOrDefault` here always return something?

Does this need a null check?

Reply:
The result is a struct: https://microsoft.github.io/Win2D/html/T_Microsoft_Graphics_Canvas_Text_CanvasFontProperty.htm

so the `.Value` should be `null`.

Reply:
What @mattleibow said. Also, I was thinking about caching the return value but it would be pointless because the resulting FontFamily objects are already being cached here.